feature-libs/order/components/order-history/order-history.component.ts

```diff
@@ -42,6 +50,11 @@ export class OrderHistoryComponent implements OnDestroy {
         if (orders?.pagination?.sort) {
           this.sortType = orders.pagination.sort;
         }
+        if (orders?.orders?.length) {
+          this.hasPONumber =
+            orders.orders[0].purchaseOrderNumber !== undefined &&
```

Comment:
We could simplify this to `orders.orders[0].purchaseOrderNumber && this.featureConfigService?.isLevel('5.1')`.

Reply:
Okay. In that case, could we simplify to `orders.orders[0].purchaseOrderNumber !== undefined && this.featureConfigService?.isLevel('5.1')`?

Reply:
Better yet, we can remove the `if` and have: 
`this.hasPONumber = orders?.orders?.[0].purchaseOrderNumber !== undefined && this.featureConfigService?.isLevel('5.1')`

Reply:
It could also be nice to add a `TODO:`about cleaning up the feature level in this equation when moving to the next major.

Reply:
done thanks~